starfish/image/_learn_transform/translation.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+from skimage.feature import register_translation
+from skimage.transform._geometric import SimilarityTransform
+
+from starfish.image._learn_transform.transforms_list import TransformsList
+from starfish.imagestack.imagestack import ImageStack
+from starfish.types import Axes, TransformType
+from starfish.util import click
+from ._base import LearnTransformBase
+
+
+class Translation(LearnTransformBase):
+
+    def __init__(self, reference_stack: ImageStack, axes: Axes, upsampling: int=1):
+        """
+        Parameters
+        ----------
+        axes:
+            The axes {r, ch, zplane} to iterate over
+        reference_stack: ImageStack
+            The target image used in skimage.feature.register_translation
+        upsampling: int
+            upsampling factor, defualt 1
+        """
+        self.upsampling = upsampling
+        self.axes = axes
+        self.reference_stack = reference_stack
+
+    def run(self, stack: ImageStack) -> TransformsList:
+        """
+        Iterate over the given axes of an ImageStack and learn the Similarity transform
+        based off the instantiated reference_image.
```

Comment:
reference_stack?  

how about:
```suggestion
        based off the reference_stack passed into :py:class:`Translation`'s constructor.
```